### PR TITLE
Mask password in loggly files.

### PR DESCRIPTION
**Pull request by darkodosenovic**
:elephant: 

Masking password in files comming to loggly

### CODE REVIEW

File: templates/21-loggly-base.conf [hunk 1]
```diff
@@ -7,5 +7,14 @@ $PrivDropToGroup adm
...
+$template LogglyFormat{{ loggly.application.tag }},"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"{{ loggly.application.tag }}\"] %$!msg%\n"
+
+if re_match($msg,'"password":"(.[^"]+)')
+then
+{
+  set $!ext = re_extract($msg,'"password":"(.[^"]+)',0,1,"");
+  set $!msg= replace($msg, $!ext, "**************");
+}
+else
+  set $!msg = $msg;
```
Comment (mvandenbeuken) [inline templates/21-loggly-base.conf]: new line

File: templates/21-loggly-base.conf [hunk 2]
```diff
@@ -7,5 +7,14 @@ $PrivDropToGroup adm
 # Setup disk assisted queues
 $WorkDirectory /var/spool/rsyslog # where to place spool files
 
-$template LogglyFormatNginx,"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"nginx\"] %msg%\n"
-$template LogglyFormat{{ loggly.application.tag }},"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"{{ loggly.application.tag }}\"] %msg%\n"
+$template LogglyFormatNginx,"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"nginx\"] %$!msg%\n"
+$template LogglyFormat{{ loggly.application.tag }},"<%pri%>%protocol-version% %timestamp:::date-rfc3339% %HOSTNAME% %app-name% %procid% %msgid% [{{ loggly.token }}@41058 tag=\"{{ loggly.application.tag }}\"] %$!msg%\n"
+
```
Comment (jbabe) [inline templates/21-loggly-base.conf]: we'll have to include the catchall template `LogglyFormat` too - maybe we just move to here for now.

Comment (jbabe) [inline templates/21-loggly-base.conf]: actually - we shouldn't be entering passwords elsewhere so this should be good to go.

**Comment by jbabe** [comment]
:elephant: :elephant: :elephant:

**Comment by darkodosenovic** [comment]
:wave:

**Comment by rubene** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**